variables.tf

```diff
@@ -16,64 +16,72 @@
 
 variable "organization_id" {
   description = "The organization id for putting the policy"
-  default     = ""
+  type        = number
+  default     = null
 }
 
 variable "folder_id" {
   description = "The folder id for putting the policy"
-  default     = ""
+  type        = number
+  default     = null
 }
 
 variable "project_id" {
   description = "The project id for putting the policy"
-  default     = ""
+  type        = string
+  default     = null
 }
 
 variable "enforce" {
   description = "If boolean constraint, whether the policy is enforced at the root; if list constraint, whether to deny all (true) or allow all"
-  default     = ""
+  type        = bool
+  default     = null
 }
 
 variable "allow" {
   description = "(Only for list constraints) List of values which should be allowed"
-  type        = "list"
+  type        = list(string)
   default     = [""]
```

Comment:
Please recheck if we still need empty string here

Reply:
We do, If we want to stick to minimal changes.

```
[11:24][user@host:~/workspace/google/terraform-google-org-policy/examples/boolean_org_exclude]$ terraform plan

Error: Missing required argument

  on main.tf line 28, in module "org-disable-serial-port-access-deny-all-with-excludes":
  28: module "org-disable-serial-port-access-deny-all-with-excludes" {

The argument "allow" is required, but no definition was found.


Error: Missing required argument

  on main.tf line 28, in module "org-disable-serial-port-access-deny-all-with-excludes":
  28: module "org-disable-serial-port-access-deny-all-with-excludes" {

The argument "deny" is required, but no definition was found.


Error: Missing required argument

  on main.tf line 28, in module "org-disable-serial-port-access-deny-all-with-excludes":
  28: module "org-disable-serial-port-access-deny-all-with-excludes" {

The argument "exclude_projects" is required, but no definition was found.
```

Reply:
It is fine, just wanted to check if it was here due to some tf0.11 limitations